test/common.sh

```diff
@@ -72,20 +76,28 @@ function deploy() {
 		--tenant "${TENANT_ID}" &>/dev/null
 
 	az account set --subscription "${SUBSCRIPTION_ID}"
+}
+
+function deploy_template() {
+	# Check pre-requisites
+	[[ ! -z "${DEPLOYMENT_NAME:-}" ]] || (echo "Must specify DEPLOYMENT_NAME" && exit -1)
```

Comment:
I figured that might be the case. I very much appreciate it, but I'm more than happy to adjust stuff on my end if you're going to cleanup how this is organized 😄.